### ISSUE TITLE
Type definitions for Typescript

### ISSUE THREAD

**Issue by ChakshuGautam** [project-sunbird/sunbird-telemetry-service#29]
Would be great if we can have types, will help speed up development and reduce bugs. Types are anyway created in the QuML player and duplicated in the Sunbird Ed portal.

@vrayulu

**Comment by santhosh-7777** [comment]
@ChakshuGautam is this one still open i would like to work on this?

**Comment by rishiii183** [comment]
hey @ChakshuGautam i would like to work this if its still open could you pls assign me?

**Comment by pallakartheekreddy** [comment]
@rishiii183 Yes, this item is still open. Please share a PR for it, and I’ll take it up from there.

**Comment by rishiii183** [comment]
@pallakartheekreddy Hi! I've opened the Pull Request for this issue: #84. 
It includes complete TypeScript type definitions for all Sunbird V3 Telemetry events, service contracts, exported package typings, and type validation unit tests. 
All checks (`npm run check-types`, `npm run lint`, and `npm test`) pass cleanly. Please review when you get a chance. Thanks!
